SchleswigGottorf/GotD782.xml

```diff
@@ -0,0 +1,84 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="GotD782" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾArgānona wǝddāse</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="SD"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt><p>Cataloguing funded by the AHRC-DFG Project
+                  'Demarginalizing medieval Africa: Images, texts, and identity
+                  in early Solomonic Ethiopia (1270-1527)', grant ref.
+                  nos. AH/V002910/1 - 448410109</p></editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0.</licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0405Gottorf_Castle_"/>
+                        <idno>D 782</idno>
+                    </msIdentifier>
+                    <msContents>
+                       <summary/>
+                       <msItem xml:id="ms_i1">
+                        <title ref="LIT1146Argano"/>
+                         <textLang mainLang="gez"/>
+                      </msItem>
+                    </msContents>
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>
+                              <material key="parchment"/>
+                           </support>
+                            <foliation>The manuscript is not foliated.</foliation>
+                         </supportDesc>
+                     </objectDesc>
+                      </physDesc>
```

Comment:
I was also wondering if Dettenberg should be mentioned in provenance

Reply:
If it is acceptable, could you please merge?